plaso/parsers/dsv_parser.py

```diff
@@ -114,6 +118,63 @@ def _CreateDictReader(self, line_reader):
         quotechar=quotechar, restkey=magic_test_string,
         restval=magic_test_string)
 
+  # pylint: disable=missing-return-type-doc
+  def _CreateLineReader(self, file_object):
+    """Returns an object that returns lines from a text file.
```

Comment:
Please change docstring.

`Returns an object that returns lines from a text file.`

=> `Creates an object that reads lines from a text file.`